jetty-server/src/main/java/org/eclipse/jetty/server/ForwardedRequestCustomizer.java

```diff
@@ -420,6 +432,12 @@ else if (forwarded._server != null)
         }
     }
 
+    protected void onError(HttpField field, Throwable t)
+    {
+        LOG.warn("Exception while processing {}", field, t);
```

Comment:
Don't warn and throw.   Do one or the other.   I think this should just be a debug here.